sdk/src/Core/Amazon.Runtime/Credentials/Internal/AnonymousAuthScheme.cs

```diff
@@ -30,5 +31,11 @@ public IIdentityResolver GetIdentityResolver(IIdentityResolverConfiguration conf
         {
             return configuration.GetIdentityResolver<AnonymousAWSCredentials>();       
         }
+
+        /// <inheritdoc/>
+        public ISigner Signer()
+        {
+            return new NullSigner();
```

Comment:
For each of these schemes should the signer be a member variable so we can reuse the same instance instead of recreating the signer for every request?